src/scripts/GameConstants.ts

```diff
@@ -658,10 +659,10 @@ namespace GameConstants {
         'Mt. Pyre',
         'Shoal Cave',
         'Seafloor Cavern',
+        'Sealed Chamber',
```

Comment:
You will also need to update the statistics for every dungeon index after this one, as it puts them all out of order.